crates/ruff_python_formatter/tests/snapshots/black_compatibility@miscellaneous__force_pyi.py.snap

```diff
@@ -89,29 +89,28 @@ def eggs() -> Union[str, int]: ...
 -def foo() -> None: ...
 +def foo() -> None:
 +    ...
-+
-+
+ 
+-class F(A, C): ...
+ 
+-def spam() -> None: ...
 +class F(A, C):
 +    ...
 +
 +
 +def spam() -> None:
```

Comment:
Hmm, we'll need into single line bodies at some point. Potentially something that our suite refactor would simplify but we shouldn't be blocked by it.l